service/routes.py

```diff
@@ -87,27 +87,27 @@ def index():
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
-@app.route("/promotions/create", methods=["POST"])
+@app.route("/promotions", methods=["POST"])
 def create_promotion():
     """
     Creates a new Promotion
     This endpoint will create a Promotion based on the data in the body that is posted
     """
     app.logger.info("Request to create a Promotion")
+    check_content_type("application/json")
     data = request.get_json()
     promotion = Promotion()
     try:
         promotion.deserialize(data)
         promotion.create()
         message = promotion.serialize()
-        location_url = url_for("get_promotion", promotion_id=promotion.promotion_id, _external=True)
-        return jsonify(message), status.HTTP_201_CREATED, {"Location": location_url}
+        # location_url = url_for("get_promotion", promotion_id=promotion.promotion_id, _external=True)
+        return jsonify(message), status.HTTP_201_CREATED
     except DataValidationError as error:
         abort(status.HTTP_400_BAD_REQUEST, str(error))
-    except Exception as error:  # pylint: disable=broad-except
+    except Exception as error:
```

Comment:
I think we should include this comment to disable the specific lint error regarding broad exceptions.

Reply:
Okay, sure